homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import "./App.css";
+import Button from "./component/button.jsx";
+import Input from "./component/todoInput";
+
+const App = () => {
+  const [todos, setTodos] = useState([
+    {
+      id: 1,
+      description: "Get out of bed",
+      deadline: "2017-09-11",
+      done: true
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+      deadline: "2017-09-10",
+      done: false
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      deadline: "2017-09-09",
+      done: false
+    }
+  ]);
+
+  const [description, setDescription] = useState("");
+  const [date, setDate] = useState("");
+
+  const handleInput = e => {
+    e.preventDefault();
+    e.target.type === "text"
+      ? setDescription(e.target.value)
+      : setDate(e.target.value);
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    const newItem = {
+      id: todos.length + 1,
```

Comment:
Using `todos.length` as value for the id of the todo is a bit dangerous. An id should always be unique, this way the same id can appear twice when adding and removing todo in a certain order.

Reply:
fixed with uuid.